server/apps/publish/archive_publish.py

```diff
@@ -318,8 +318,8 @@ def publish(self, doc, updates, target_media_type=None):
 
                 # Step 2(a)(i)
                 subscribers = list(get_resource_service('subscribers').get(req=None, lookup=None))
-                recipients = [s['email'] for s in subscribers]
-                send_article_killed_email(doc, recipients, queued_items[0].get('completed_at'))
+                recipients = [s.get('email') for s in subscribers]
```

Comment:
This can be a generator, since it's only used as an iterable passed to `filter()` in the subsequent line, right?